drivers/scheduler/rke/rke.go

```diff
@@ -184,6 +184,109 @@ func (r *Rancher) GetProjectID(projectName string) (string, error) {
 	return projectId, fmt.Errorf("no project matching the given projectName %s was found", projectName)
 }
 
+// CreateUserForRancherProject Creates a rancher user and adds the user to the project
+func (r *Rancher) CreateUserForRancherProject(projectName string, userName string, userPassword string) (string, error) {
```

Comment:
nit: You can call it `username` and `password`

Reply:
Addressed it.